openlr/openlr_stat/openlr_stat.cpp

```diff
@@ -27,35 +33,14 @@ void LoadIndex(Index & index)
   auto const latestVersion = numeric_limits<uint64_t>::max();
   FindAllLocalMapsInDirectoryAndCleanup(dir, 0 /* version */, latestVersion, localFiles);
 
-  Platform::TFilesWithType fwts;
```

Comment:
Will this work with structure like this
- data
  - 161010
    - mwm1
    - mwm2
    - ...
?